tests/console/consoletest_setup.pm

```diff
@@ -62,7 +62,8 @@ sub run {
     # init
     check_console_font;
 
-    type_string "chown $username /dev/$serialdev\n";
+    # This enables user to access serial in case of reboot system happened during console tests
+    script_run "echo 'chown $username /dev/$serialdev' >> /etc/bash.bashrc.local";
```

Comment:
bad idea - users do not have permission to change the dev ownership - and thus all user logins / terms warn about it

e.g.:

* https://openqa.opensuse.org/tests/477682#step/mate_terminal/5
* https://openqa.opensuse.org/tests/477382#step/gnome_terminal/5

Reply:
IMHO not a good approach and with an impact far beyond what you might think. I suggest to revert this special part

Reply:
ah, i see, didn't consider the cases in user logins :(
thanks for correct